CPAC/pipeline/cpac_pipeline.py

```diff
@@ -845,7 +846,10 @@ def build_anat_preproc_stack(rpool, cfg, pipeline_blocks=None):
         pipeline_blocks += anat_init_blocks
 
     if not rpool.check_rpool('freesurfer-subject-dir'):
-        pipeline_blocks += [freesurfer_preproc]
+        pipeline_blocks += [freesurfer_reconall]
+        pipeline_blocks += [freesurfer_postproc]
+    else:
+	pipeline_blocks += [freesurfer_postproc]
```

Comment:
```suggestion
        pipeline_blocks += [freesurfer_postproc]
```

(it's 1 tab instead of 8 spaces right now)

Reply:
Fixed in 8df8c6c7ad85ff0413695c35f0a029ee01c90df4